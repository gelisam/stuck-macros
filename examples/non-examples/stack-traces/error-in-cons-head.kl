#lang "prelude.kl"

(import "lambda-case.kl")

-- TODO: DYG: how to test the pairs?
(define fail
  (lambda (thing)
    (car '(1 2 'something-else))))

(example (fail 3))