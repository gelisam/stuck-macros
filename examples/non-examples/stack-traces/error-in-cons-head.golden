Unknown: #[error-in-cons-head.kl:8.6-8.9]<car>
